src/components/sharing/index.js

```diff
@@ -45,31 +43,33 @@ import isQueryLoading from "../utils/isQueryLoading";
 import GridLoading from "../utils/GridLoading";
 import ids from "./ids";
 import SharedItem from "./SharedItem";
-import Identity from "../data/Identity";
 import SubjectSearchField from "./SubjectSearchField";
 import { useUserProfile } from "contexts/userProfile";
 import { useTranslation } from "i18n";
-import SharingPermissionSelector from "components/sharing/SharingPermissionSelector";
 import Permissions from "components/models/Permissions";
 import {
     doSharingUpdates,
     GET_PERMISSIONS_QUERY_KEY,
     getPermissions,
 } from "../../serviceFacades/sharing";
-import withErrorAnnouncer from "components/utils/error/withErrorAnnouncer";
 import styles from "./styles";
+import UserTable from "./UserTable";
+import ErrorTypography from "../utils/error/ErrorTypography";
+import DEErrorDialog from "../utils/error/DEErrorDialog";
```

Comment:
This can be updated to `components/utils/error/...` for consistency. Think we agreed to move towards absolute imports for some performance improvements 🤔.